hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/OMClientRequest.java

```diff
@@ -127,7 +127,14 @@ public OMRequest getOmRequest() {
     }
 
     if (remoteAddress != null) {
-      userInfo.setHostName(remoteAddress.getHostName());
+      OzoneManagerProtocolProtos.UserInfo userInfoInProtocol = getOmRequest()
```

Comment:
We don't need toBuilder here, we can get UserInfo by calling getUserInfo()

Like `getOmRequest().getUserInfo()`